### PR TITLE
*: add six documents of TiDB Lightning

### PR DESCRIPTION
**Pull request by lilin90**
@kennytm @CaitinChen PTAL
Related PR: https://github.com/pingcap/tidb-lightning/pull/80

### CODE REVIEW

**Comment by kennytm** [comment]
cc @GregoryIan. 

Since we'll also host the docs on https://github.com/pingcap/tidb-tools/tree/docs/docs, do you think it's a good idea to create a copy on the official site here, or should the official site just be a 1-page description that links to tidb-tools docs?

**Comment by lilin90** [comment]
@kennytm @GregoryIan It is a good idea to maintain the documents only in one place. 

I think it'd be better to keep the complete detailed contents on the official website. Because some users need to download the PDF version of the whole documentation. This will be more user-friendly. 

Perhaps we can add the link of each document (that in pingcap/docs/tools, or the links on the official website if you like) to the "tidb-tools" repo.

**Comment by lilin90** [comment]
@CaitinChen @kennytm @GregoryIan PTAL

File: tools/lightning-checkpoints.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+title: TiDB Lightning 断点续传
+category: tools
+---
+
+# TiDB Lightning 断点续传
```
Comment (kennytm) [inline tools/lightning-checkpoints.md]:
```suggestion
# TiDB-Lightning 断点续传
```

According to https://github.com/pingcap/tidb-lightning/pull/80#discussion_r232165326 we should hyphenate it

**Comment by lilin90** [comment]
@kennytm Thanks! PTAL again

**Comment by lilin90** [comment]
@kennytm All "TiDB-Lightning"s have been updated. I'll merge it soon :)

**Pull request merged**